src/mame/seta/seta.cpp

```diff
@@ -10037,56 +10037,56 @@ ROM_START( msgundam1 )
 
-	ROM_REGION( 0x080000, "gfx3", 0 )   /* Layer 2, on lower board, identical to the original but split */
+	ROM_REGION( 0x080000, "gfx3", 0 )   // Layer 2, on lower board, identical to the original but split
 	ROM_LOAD16_BYTE( "d-8.bin", 0x000001, 0x040000, CRC(a03c8345) SHA1(d3c3f0045ebb3d82d82432c212db4a801cb53b60) ) // 27c020
 	ROM_LOAD16_BYTE( "d-9.bin", 0x000000, 0x040000, CRC(cfd47024) SHA1(ab4fbaf258d2694407dd6c896f45d69821ccc408) ) // 27c020
 
-	ROM_REGION( 0x100000, "x1snd", 0 )  /* Samples, on lower board */
+	ROM_REGION( 0x100000, "x1snd", 0 )  // Samples, on lower board
 	ROM_LOAD( "27c8001-4.bin", 0x000000, 0x100000, CRC(b965f07c) SHA1(ff7827cc80655465ffbb732d55ba81f21f51a5ca) )  // identical to the original
 ROM_END
 
 ROM_START( oisipuzl )
-	ROM_REGION( 0x180000, "maincpu", 0 )        /* 68000 Code */
+	ROM_REGION( 0x180000, "maincpu", 0 )        // 68000 Code
 	ROM_LOAD16_WORD_SWAP( "ss1u200.v10", 0x000000, 0x080000, CRC(f5e53baf) SHA1(057e8b35bc6f65634685b5d0cf38e12f2e62d72c) )
-	/* Gap of 0x80000 bytes */
+	// Gap of 0x80000 bytes
 	ROM_LOAD16_WORD_SWAP( "ss1u201.v10", 0x100000, 0x080000, CRC(7a7ff5ae) SHA1(9e4da7ecc4d833c3ba4ddc6e5870fad53b9b2d2b) )
 
-	ROM_REGION( 0x200000, "gfx1", ROMREGION_INVERT )    /* Sprites */
+	ROM_REGION( 0x200000, "gfx1", ROMREGION_INVERT )    // Sprites
 	ROM_LOAD( "ss1u306.v10", 0x000000, 0x080000, CRC(ce43a754) SHA1(3991042678badafee716b084c1768a794f144b1e) )
 	ROM_LOAD( "ss1u307.v10", 0x080000, 0x080000, CRC(2170b7ec) SHA1(c9f3d12646d4e877bc2b656f977e21d927f241f6) )
 	ROM_LOAD( "ss1u304.v10", 0x100000, 0x080000, CRC(546ab541) SHA1(aa96a79e3b0ba71f5e0fbb15e190d219630c2ba3) )
 	ROM_LOAD( "ss1u305.v10", 0x180000, 0x080000, CRC(2a33e08b) SHA1(780cfe44a4d57b254bd0cfae8727dc77358027a8) )
 
-	ROM_REGION( 0x100000, "gfx2", 0 )   /* Layer 1 */
+	ROM_REGION( 0x100000, "gfx2", 0 )   // Layer 1
 	ROM_LOAD16_WORD_SWAP( "ss1u23.v10",  0x000000, 0x080000, CRC(9fa60901) SHA1(3d42e4174ad566b6eeb488c7a4c51db9c1fef7af) )
 	ROM_LOAD16_WORD_SWAP( "ss1u24.v10",  0x080000, 0x080000, CRC(c10eb4b3) SHA1(70a82a750b1d9c849cd92d4f73769bbf5962c771) )
 
-	ROM_REGION( 0x080000, "gfx3", 0 )   /* Layer 2 */
+	ROM_REGION( 0x080000, "gfx3", 0 )   // Layer 2
 	ROM_LOAD16_WORD_SWAP( "ss1u25.v10",  0x000000, 0x080000, CRC(56840728) SHA1(db61539fd84f0de35ee2077238ba3646c4960cc6) )
 
-	ROM_REGION( 0x100000, "x1snd", 0 )  /* Samples */
+	ROM_REGION( 0x100000, "x1snd", 0 )  // Samples
 	ROM_LOAD( "ss1u26.v10", 0x000000, 0x080000, CRC(d452336b) SHA1(d3bf3cb383c40911758a60546f121c48087868e3) )
 	ROM_LOAD( "ss1u27.v10", 0x080000, 0x080000, CRC(17fe921d) SHA1(7fc176b8eefad4f2b8532bfe62e7852d2be185ca) )
 ROM_END
 
-ROM_START( triplfun ) /* the program fails its self-check but thats probably because its a bootleg, it does the same on the real board */
+ROM_START( triplfun )
```

Comment:
Is the comment here incorrect?

Reply:
There no fail on self tests on this version